tests/Controller/CRUDControllerTest.php

```diff
@@ -74,6 +75,10 @@
  */
 class CRUDControllerTest extends TestCase
 {
+    use ExpectDeprecationTrait {
+        expectDeprecation as traitExpectDeprecation;
```

Comment:
Alias needed for: 

```Fatal error: Access level to Symfony\Bridge\PhpUnit\ExpectDeprecationTrait::expectDeprecation() must be public (as in class PHPUnit\Framework\TestCase) in /home/travis/build/sonata-project/SonataAdminBundle/tests/Controller/CRUDControllerTest.php on line 76```

https://travis-ci.org/github/sonata-project/SonataAdminBundle/jobs/711365626